Sources/Sentry/SentryTracer.m

```diff
@@ -28,6 +28,11 @@ - (NSArray *)children;
  * A lock to coordinate child manipulation.
  */
 - (NSLock *)childrenLock;
+
+/**
+ * List of children. For testing purpose.
+ */
+- (NSArray *)spans;
```

Comment:
`l`: Can we please use generics here to make the code easier to read. Same applies for `- (NSArray *)children;`
```suggestion
- (NSArray<id<SentrySpan>> *)spans;
```